django-backend/fecfiler/web_services/tasks.py

```diff
@@ -229,16 +230,14 @@ def poll_for_fec_response(submission_id, submission_type_key, submission_name):
 
         submission.fecfile_polling_attempts += 1
         logger.info(f"Polling status for {submission.fec_submission_id}.")
-        logger.info(
-            f"Status: {submission.fec_status}, Message: {submission.fec_message}"
-        )
+        logger.info(f"Status: {submission.fec_status}, Message: {submission.fec_message}")
         logger.info(
             f"""Submission Polling - Attempt {
                 submission.fecfile_polling_attempts
             } / {EFO_POLLING_MAX_ATTEMPTS}"""
         )
         status_response_string = submitter.poll_status(
-            submission.fec_batch_id, submission.fec_submission_id
+            getattr(submission, "fec_batch_id", None), submission.fec_submission_id
```

Comment:
I think there's a better way to make poll_status generic.  Rather than having it take two different ids, one per type, lets have it take one id, and decide which to use in this method in the line before.

```
poll_id = getattr(submission, "fec_batch_id", None) or getattr(submission, "fec_submission_id", None)
status_response_string = submitter.poll_status(poll_id)
```

Reply:
good catch.  @toddlees and I discussed and I updated the poll_status function to take a submission object to allow it to be generic as well as flexible with batch_id.